src/components/home.js

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect } from 'react';
+import { useSelector, shallowEqual, useDispatch } from 'react-redux';
+import { fetchCountry } from '../redux/countries/fetchcountries';
+import AllCountry from './allCountries';
+import europe from '../images/europe.png';
+import '../App.css';
+
+let retriveData = true;
+const HomePage = () => {
```

Comment:
- [Optional] We are expected to follow the design guidelines for this project including typography, and layout. Kindly compare the two screenshots below and make the requested changes.
- It is important to note that having a nice design increases your chances for a job as recruiters will be checking your app.

| Yours | Guidelines |
| --- | --- |
| ![Screenshot from 2022-12-07 14-03-40](https://user-images.githubusercontent.com/63087491/206187419-f22452ae-8d03-40df-b32f-050c26007fbf.png) | ![Screenshot from 2022-12-02 19-27-09](https://user-images.githubusercontent.com/63087491/205361582-9ef96d0e-f5fe-4e80-a3f4-81f168e07813.png) |